tests/rados/test_osd_inprogress_rebalance.py

```diff
@@ -6,7 +6,7 @@
 from tests.rados.rados_test_util import (
     create_pools,
     get_device_path,
-    wait_for_device,
+    wait_for_device_rados,
```

Comment:
is the plan to do away with older implementation and use the new method? and replace each call one by one?

Reply:
That's correct.
The old implementation could have been changed if the method was only being used in rados test cases, however, few rbd and  cephfs tests are also utilizing the same module and I did not want to affect them.
If the new implementation works flawlessly for a series of runs, we can make it the default.